WORKSPACE

```diff
@@ -104,6 +128,14 @@ filegroup(
     name = "code_proto",
     srcs = [ "google/rpc/code.proto" ],
 )
+
+filegroup(
+    name = "api_protos",
```

Comment:
Went with `metric_proto`, `label_proto` to keep the names similar to `code`/`status_proto` immediately above.